tests/js/spec/components/smartSearchBar.spec.jsx

```diff
@@ -110,7 +110,7 @@ describe('SmartSearchBar', function () {
     await tick();
 
     expect(searchBar.find('textarea').props().value).toEqual(
-      'device:"this is filled with spaces"'
+      'device:"this is filled with spaces" '
```

Comment:
I believe this is more correct (e.g. if you autocomplete this at sentry.io, it adds a space) cc @EvanPurkhiser